nala/blocks/smoke-test/smoke.page.js

```diff
@@ -0,0 +1,69 @@
+export default class SmokeTest {
+  constructor(page) {
+    this.page = page;
+    this.joinNowButton = page.locator('text="Join Now"');
+    this.signInButton = page.locator('.feds-signIn');
+    this.GNav = page.locator('.feds-topnav');
+    this.profileIcon = page.locator('.feds-profile-button');
+    this.priceList = page.locator('a[href="https://partners.stage.adobe.com/na/channelpartners/home/pricelists/"]');
```

Comment:
I don't think this is this going to work on prod, right?
We should find a way that the smoke tests pass on prod and on stage